specifications/config.json

```diff
@@ -0,0 +1,127 @@
+{
+  "client": {
+    "apiKey":{
+      "file":"",
+      "id":"",
+      "secret":""
+    },
+    "cacheManager":{
+      "defaultTtl": 300,
+      "defaultTti": 300,
+      "caches": {
+        "account": {
+          "ttl": 300,
+          "tti": 300
+        }
+      }
+    },
+    "baseUrl": "https://api.stormpath.com/v1",
+    "connectionTimeout": 30,
+    "authenticationScheme": "SAUTHC1", // or "BASIC"
+    "proxy":{
+      "port": "",
+      "host": "",
+      "username": "",
+      "password": ""
+    }
+  },
+  "application": {
+    "name": "test",
+    "href": "https://..."
+  },
+  "web": {
+    "oauth2":{
+      "enabled": false,
+      "uri": "/oauth/token",
+      "password":{
```

Comment:
That's true for `password`, as we will be issuing those from our new OAuth endpoints (and their configuration is controlled by the application's OAuth policy).

`client_credentials` still needs to be configured locally, as that grant flow (using api keys on accounts) is still happening only on the local SDK